packages/upswyng-web/src/components/HotlineCard.tsx

```diff
@@ -17,24 +18,31 @@ interface Props {
 const HotlineCard = ({ name, description, phone }: Props) => {
   const [hideText, setHideText] = React.useState(true);
   return (
-    <ListItem button>
+    <ListItem>
       <ListItemText
-        onClick={() => initiatePhoneCall(phone)}
         primary={name}
         secondary={
           <>
-            <Typography>{phone}</Typography>
+            <Link
+              underline="always"
+              onClick={() => initiatePhoneCall(phone)}
```

Comment:
```suggestion
              href={`tel:${phone}`}
```
Could we use the `tel:` schema here instead?

Reply:
Totally - we just have to include this regex so that it works on all browsers `tel:${phoneNumber.replace(/[-]/g, "")}`

Reply:
Totally - we just have to include this regex so that it works on all browsers tel:${phoneNumber.replace(/[-]/g, "")}